easybuild/easyconfigs/s/Spark/Spark-2.4.5-intel-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,56 @@
+easyblock = 'Tarball'
+
+name = 'Spark'
+version = '2.4.5'
+versionsuffix = '-Python-%(pyver)s'
```

Comment:
@Darkless012 Since `Java/11` is the preferred `Java` version in the `2019b` toolchains, we should also add `-Java-%(javaver)s` to the `versionsuffix` here (and let that trickle up into `hyperopt` as well)...